src/getting-started.md

```diff
@@ -171,6 +171,10 @@ incredibly helpful:
 
 See ["How to build and run the compiler"](./building/how-to-build-and-run.md).
 
+## How do i contribute to std?
```

Comment:
I would change the heading, to match the others
```suggestion
## Contributing to std (standard library)
```
I would also move it to before "Contributing code to other Rust projects", because std is more central than those other projects.

Reply:
done.